### PR TITLE
chore(readme): add info on linting other file types

### PR DESCRIPTION
**Pull request by trusktr**
I was scratching my head playing with `eslintrc` trying to figure why it wouldn't work, then I realized I had to add it to the scopes list when I stumbled on https://github.com/AtomLinter/linter-eslint/issues/1122

### CODE REVIEW

**Comment by skylize** [comment]
Thanks for helping out! We can always benefit from better documentation. But this issue was fixed on Atom's side by changing Tree Sitter to use the old `source.js` scope name. 

Update to the newest [Release 1.32](http://blog.atom.io/2018/10/23/atom-1-32.html), which includes activating Tree Sitter by default and has no need for adding the `javascript` scope name that was required during the experimental stages.

**Comment by skylize** [comment]
Sorry. I just actually read the commits, and realized I misunderstood what you were adding. 🤦‍♂️  because of the issue you linked to.

File: README.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ installed
 for you. If you are using an alternative `linter-*` consumer feel free
 to disable the `linter` package.
 
+> **Important!** If you wish to lint other files besides JavaScript files (f.e. `.js`), you need to add them to `List of scopes to run ESLint on` option in linter-eslint settings. For example, to lint TypeScript files, you'll need to add `source.ts` to the list.
```
Comment (skylize) [inline README.md]: Definitely do *not* think this qualifies for a bold "Important". This is intended for ESLint, which is in turn intended for JavaScript. Other languages are sort of an edge case. (Typescript has its own linter.)

Comment (skylize) [inline README.md]: This should go in A Note About Settings, not in Installation.

Comment (Arcanemagus) [inline README.md]:
```suggestion
> **Important!** If you wish to lint files besides JavaScript, you'll need to add them to the `List of scopes to run ESLint on` option in `linter-eslint`'s settings. For example, to lint TypeScript files, you'll need to add `source.ts` to the list.
```

Comment (skylize) [inline README.md]:
```suggestion
If you wish to lint files in JavaScript-derivative languages (like Typescript, Flow) with ESLint, you must add the scope name for that grammar to the `List of scopes to run ESLint on` option in `linter-eslint Settings`. For example, to lint TypeScript files, add `source.ts` to the list.
```

**Comment by dakotahp** [comment]
@trusktr This PR is the only way I figured out how to get ESLint to run on TypeScript files. If you make these suggested changes then they can approve it for merging. Took me weeks to figure out how to do this and it's a shame that the readme doesn't already document this.

**Review by UziTech — APPROVED** [review]

**Pull request merged**